front/app/modules/commercial/customizable_homepage_banner/admin/LayoutSetting.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+
+// components
+import { Label, Radio, Box } from 'cl2-component-library';
+import { SectionField } from 'components/admin/Section';
+
+import Layout1Active from './layout_previews/layout1_active.jpg';
+import Layout2Active from './layout_previews/layout2_active.jpg';
+import Layout3Active from './layout_previews/layout3_active.jpg';
+import Layout1Inactive from './layout_previews/layout1_inactive.jpg';
```

Comment:
Isn't it possible with modern CSS to apply a black-and-white filter to an image? If the image changes, we won't need to convert it again.

Reply:
Good suggestion, didn't think of that. I'll try that out.